cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -46,7 +46,7 @@ public void testCreateUserTask() {
                                                           100, new MockTime(), mockUUIDGenerator);
     // test-case: create user-task based on request and get future
     OperationFuture future1 =
-        userTaskManager.getOrCreateUserTask(mockHttpServletRequest1, mockHttpServletResponse, uuid -> future, 0).get(0);
+        userTaskManager.getOrCreateUserTask(mockHttpServletRequest1, mockHttpServletResponse, uuid -> future, 0, true).get(0);
```

Comment:
This test class seems to be limited to async tasks -- can we add test cases for sync tasks?

Reply:
Added test-case: for sync task, UserTaskManager does not create mapping between request URL and UUID. This is the only difference between sync and async request as far as `UserTaskManager.getOrCreateUserTask` is concerned, so the same test cases will cover sync task.

Reply:
@alexwang08 Do we allow users to send UUID in header? If not we should let them send UUID in request and associate that UUID with the request.

Reply:
We allow user to send UUID in header and associated the UUID to request